.coveragerc

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+[report]
+exclude_lines =
+    pragma: no cover
+    if __name__ == .__main__.:
+    if __name__ == "__main__" :
+
+[run]
+branch = True
+omit = */__init__.py
+       mms/tests/*
+       mms/utils/model_server_error_codes.py
+       mms/utils/timeit_decorator.py
+       mms/storage.py
+       mms/metrics/system_metrics.py
+       mms/utils/mxnet/*
+       mms/examples/metric_push_example.py
+       mms/model_service/*
```

Comment:
Missing newline at end of file.  https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file might help

Reply:
Done. Thanks! 8dfa551a22b89525d9f9b66bc906219c6d9aed94